src/app/public/src/modules/shared/omnibar-adapter.service.ts

```diff
@@ -7,9 +7,13 @@ export class StacheOmnibarAdapterService {
   private static readonly HAS_OMNIBAR_CLASS_NAME: string = 'stache-omnibar-enabled';
   public static readonly EXPECTED_OMNIBAR_HEIGHT: number = 50;
   private element: any = this.windowRef.nativeWindow.document.querySelector('.sky-omnibar-iframe');
+  private configured: boolean = false;
 
-  constructor(private windowRef: StacheWindowRef) {
-    if (this.omnibarEnabled()) {
+  constructor(private windowRef: StacheWindowRef) { }
+
+  public checkForOmnibar(): void {
+    if (this.omnibarEnabled() && !this.configured) {
```

Comment:
LGTM. I would have personally rewired this class to use `Renderer2` to add the class name since it would only ever add the class once (and you'd then avoid the `this.configured` check), but I know you're spent for time.

Reply:
Just by way of demonstration:

```
import { Injectable, Renderer2, RendererFactory2 } from '@angular/core';

import { StacheWindowRef } from './window-ref';

@Injectable()
export class StacheOmnibarAdapterService {
  private renderer: Renderer2;

  constructor(
    private rendererFactory: RendererFactory2,
    private windowRef: StacheWindowRef
  ) {
    this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
  }

  public checkForOmnibar(): void {
    if (this.omnibarEnabled()) {
      this.applyClassToBody();
    }
  }

  public getHeight(): number {
    return (this.omnibarEnabled()) ? 50 : 0;
  }

  public omnibarEnabled(): boolean {
    return !!this.windowRef.nativeWindow.document.querySelector('.sky-omnibar-iframe');
  }

  private applyClassToBody(): void {
    this.renderer.addClass(
      this.windowRef.nativeWindow.document.body,
      'stache-omnibar-enabled'
    );
  }
}
```

But never mind; merge away!

Reply:
I will implement this :) I totally should have thought of that.